src/Files.App/Files.App.csproj

```diff
@@ -64,7 +64,7 @@
         <Content Remove="Assets\FilesHome.png" />
     </ItemGroup>
     <ItemGroup>
-      <None Remove="UserControls\OpacityIcon.xaml" />
+        <None Remove="UserControls\OpacityIcon.xaml" />
```

Comment:
This file should be set to spaces.

Reply:
Something looks off, I recommend reverting whatever changes were made to the file.

Reply:
Sure.